hivemq4/dns-image/config-dns.xml

```diff
@@ -31,7 +31,7 @@
         <enabled>true</enabled>
         <discovery>
             <extension>
-                <reload-interval>${HIVEMQ_DNS_DISCOVERY_INTERVAL}</reload-interval>
```

Comment:
Why did you remove this?

Reply:
This tag does actually not exist in the hivemq config.

Reply:
I think the HIVEMQ_DNS_DISCOVERY_INTERVAL should be removed from the documentation as well then, the information is on the extension documentation.

Reply:
I don't think this should go into this PR, as we also have other env variables for the dns-discovery-extension only in the README, which then should be removed as well and replaced by a link to the dns-cluster-discovery-extension.
Also, the Dockerfile currently overrides the default of 30 for the interval with 31, so I would not want to introduce a behavior change with this.